autoload/nvim_treesitter.vim

```diff
@@ -0,0 +1,16 @@
+function! s:visual_node(node_range)
+  let [l:cursor_start, l:cursor_end] = a:node_range
+  if !empty(l:cursor_start) && !empty(l:cursor_end)
+    call cursor(l:cursor_start[0]+1, l:cursor_start[1]+1)
+    normal v
```

Comment:
i understood this was the issue you couldn't write this as lua ? did you try `api.nvim_command('normal v')` ?

Reply:
also what is the purpose of this autoload file exactly ?